src/ucp/core/ucp_worker.h

```diff
@@ -29,6 +29,10 @@
 #define UCP_WORKER_HEADROOM_PRIV_SIZE 32
 
 
+/* Used as invalidated value */
+#define UCP_WORKER_PTR_KEY_INVALID      0
```

Comment:
why is this a worker and not 'ep' key?

Reply:
because the map is placed on worker, probably better to rename like UCP_WORKER_PTR_MAP_KEY_INVALID, WDYT?

Reply:
yes, better.